ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/clientlibs/site/js/datepickerview.js

```diff
@@ -54,7 +54,11 @@
             super.setModel(model);
             this.widget.addEventListener('blur', (e) => {
                 this._model.value = e.target.value;
-            })
+                this.setInactive();
```

Comment:
I think form object should store this state. `activeField` and `activePanel`

Reply:
Do we have a usecase to maintain ui state?

Reply:
Hide show of next prev buttons based on activeField/activePanel

Reply:
ok, will add the state.

Reply:
I meant the model should keep the activeField property as well.

Reply:
There should be certain rules which should be governed via guidebridgeAPI, rather than polluting model with ui state. For example form container view has the relevant information of all views. guideBridge should have a reference to this form container view. 
So our rule should be if guideBridge.isInsideActiveContainer -> parameter as field. @vdua WDT?